ivy/functional/ivy/general.py

```diff
@@ -2467,26 +2469,81 @@ def stable_divide(
 @inputs_to_native_arrays
 @handle_nestable
 def stable_pow(
-    base: Union[Number, ivy.Array, ivy.NativeArray],
-    exponent: Union[Number, ivy.Array, ivy.NativeArray],
-    min_base: float = None,
-) -> Any:
+    base: Union[Number, ivy.Array, ivy.NativeArray, ivy.Container],
```

Comment:
Please don't update the typehints here, also you don't need to add `ivy.Container` to the typehints because every nestable function supports an `ivy.Container` in any or all the arguments, which need not be specified in the functional version.